### ISSUE TITLE
Suggestion: make iterative  PR-fixes instead of big PR dumps.

### ISSUE THREAD

**Issue by janderssonse** [eu-digital-identity-wallet/eudi-srv-web-issuing-eudiw-py#67]
For someone following the project it becomes hard to follow big PR's like https://github.com/eu-digital-identity-wallet/eudi-srv-web-issuing-eudiw-py/pull/64. This also makes forks with custom changes hard to keep in sync,  with many many fixes coming at once.  Would it be possible to keep a more iterative and expected workflow i.e to merge the PR's as they happen in time? That way, anyone forking or following the project can keep their forks updated without much hazzle. Thanks.

**Comment by LuisPereira23** [comment]
@janderssonse Appreciate the feedback. That’s a good point about keeping forks updated. We’ll work on merging PRs more consistently to make things easier for everyone involved.
